content/en/docs/developer_guide/project/configuration/project_configuration.md

```diff
@@ -23,7 +23,7 @@ The following topics describe the project configuration settings.
 
 | Key | Type | Default | Description |
 | --- | --- | --- | --- |
-| allowedHosts | Array<String> | \- | Restricts access to the Admin Console to the specified hosts. |
+| allowedHosts | Array | \- | Restricts access to the Admin Console to the specified hosts. |
```

Comment:
**question:** Why was this removed? It is a valid value and reflects what is accepted for the allowedHosts.

Reply:
When you look at the published page, you only see Array and not Array<String> so I removed it thinking it was in error. If it needs to be Array<String> then <String> needs to be enclosed with back ticks. Do you want me to update the PR with Array`<String>` ? Also, how were the Markdown checks passing in the PR with <String> which looks like HTML which should not be allowed?